pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/errorprone/AvoidDuplicateLiteralsRule.java

```diff
@@ -116,17 +115,13 @@ public Object visit(ASTCompilationUnit node, Object data) {
             exceptions = p.parse(getProperty(EXCEPTION_LIST_DESCRIPTOR));
         } else if (getProperty(EXCEPTION_FILE_DESCRIPTOR) != null) {
             exceptions = new HashSet<>();
-            LineNumberReader reader = null;
-            try {
-                reader = getLineReader();
+            try (LineNumberReader reader = getLineReader()) {
                 String line;
                 while ((line = reader.readLine()) != null) {
                     exceptions.add(line);
                 }
             } catch (IOException ioe) {
                 ioe.printStackTrace();
```

Comment:
Could you change this printStackTrace please? We should just rethrow it as `throw new RuntimeException(ioe);` since if we get an error here, it doesn't make sense to continue with that rule at all, so we should simply abort.

Reply:
Done.